modules/v1/users/resources/UserResource.php

```diff
@@ -189,4 +197,35 @@ public function updateUserDepartments($data)
             }
         }
     }
+
+    /**
+     * Before save change invitation status from register to complete
+     *
+     * @param $insert
+     * @return bool|void
+     * @throws ValidationException
+     * @throws \Exception
+     */
+    public function beforeSave($insert)
+    {
+        $oldStatus = ArrayHelper::getValue($this->oldAttributes, 'status');
+
+        if ($oldStatus == self::STATUS_INACTIVE && $this->status == self::STATUS_ACTIVE && $this->invitation) {
+            if ($this->invitation->status == Invitation::STATUS_REGISTERED) {
```

Comment:
@skavt Instead of writing 2nd if statement you can join it with the first one: 
```
$oldStatus == self::STATUS_INACTIVE && $this->status == self::STATUS_ACTIVE && $this->invitation && $this->invitation->status == Invitation::STATUS_REGISTERED
```